backend/api/serializers.py

```diff
@@ -11,4 +11,9 @@
 class PostSerializer(serializers.ModelSerializer):
   class Meta:
     model = post.Post
-    fields = ['title', 'author_id', 'description', 'content', 'contentType', 'visibility']
\ No newline at end of file
+    fields = ['title', 'author_id', 'description', 'content', 'contentType', 'visibility']
+
+class SignupSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = signupRequest.Signup_Request
+    fields = ['username','password','git_url']
```

Comment:
If you take a look at signup_request model in the master branch, it has 5 fields. You might want to add displayName and host.

Reply:
Added in the latest commit